tests/test_knowledge_panels.py

```diff
@@ -95,20 +95,72 @@ def test_hunger_game_kp_label_with_value():
     }
 
 
-class MockResponse:
-    def json(self):
-        return {
-            "data-quality": {
-                "elements": [
-                    {"element_type": "text", "total_issues": 0, "text_element_1": {}}
+def test_data_quality_kp_country_only(monkeypatch):
+    class MockResponse(object):
```

Comment:
The MockResponse pattern is repeated a lot, and you could factorize it in a function that just take the json content.

It could be in a test_utils.py

Something like:

```python
class MockResponse:
    def __init__(self, json_content):
        self.json_content = json_content
    def json(self):
        return self.json_content
        
def mock_get_factory(target_url, json_content):
    """generate a mock to patch request.get with a json response"""
    # this is the function we will return
    def mock_get(url):
        assert url == target_url
        return MockResponse(json_content)
    return mock_get
```

And in your test you just have to call:

```
expected_url = "https://xxxx"
expected_json = {...}
monkeypatch.setattr(requests, "get", mock_get_factory(expected_url, expected_json)
```